documentation/docs/cheatsheets/client.md

```diff
@@ -22,5 +22,24 @@
 | Command | Description |
 |-------- | ----------- |
 | `bounty --register` | Register to participate in for liquidity bounties
+| `bounty --list` | See a list of active bounties.
+| `bounty --restore-id` | If you lost your configuration file, this is the command to restore it.
 | `bounty --status` | See your accumulated rewards
 | `bounty --terms` | See the terms & conditions
+
+## Keyboard shortcuts
+| Keyboard Combo | Command | Description |
+|-------- | ----------- | ----------- |
+| `Double CTRL + C` | Exit | Press `CTRL + C` twice to exit the bot
+| `CTRL + S` | Status | Show bot status
+| `CTRL + F` | *Search / Hide Search | Toggle search in log pane
+| `CTRL + A` | Select All | Select all text in input pane [Used for text edit in Input pane only] 
+| `CTRL + Z` | Undo | Undo action in input pane [Used for text edit in Input pane only] 
+| `Single CTRL + C` | Copy | Copy text [Used for text edit in Input pane only] 
+| `CTRL + V` | Paste | Paste text [Used for text edit in Input pane only] 
+
+***Note about search:*** 
```

Comment:
Should there be a description of what this does?  Searches the log pane.  Kind of hidden in keyboard shortcuts.